ccm_web/client/src/models.ts

```diff
@@ -0,0 +1,3 @@
+export interface HelloMessageData {
+  message: string
```

Comment:
what is purpose of the file? and how it will be used for future? A short comment would be use full.

Reply:
I'll add a comment. `models` (it could be called something else) would be used to define interfaces for common entity types/objects in the application. For instance, in the future we could define `Section`, `Course`, or `Feature` objects that could be used throughout the application. This will hopefully reduce some duplication, promote common patterns.

Reply:
This file seems to be synonymous to `Config.ts`, isn't it? If Yes the client and server can have same names like `ConfigClient.ts` and `ConfigServer.ts`. If No adding a comment would help

Reply:
Mmm, not exactly, although the `models.ts` might define interfaces for some configuration data. This will also include interfaces for data originating from the Canvas API (e.g. `Section`) and other places in our application (like a database table).